components/skus/browser/resources/skus_internals.tsx

```diff
@@ -29,9 +29,22 @@ const StateContainer = styled.div`
 
 const API = SkusInternalsMojo.SkusInternals.getRemote()
 
+interface CreateOrderFromReceiptInputFields {
+  domain: string
+  receipt: string
+  order_id: string
+}
+
+const defaultState: CreateOrderFromReceiptInputFields = {
+  domain: 'vpn.brave.com',
```

Comment:
I don't think this process is supposed to support VPN at this point. We would like to eventually move to it, but not now.